manifests/init.pp

```diff
@@ -60,18 +70,23 @@
     }
 
     package { 'openssh':
-        ensure            => $ensure,
+        ensure            => $manage_package,
```

Comment:
Here $manage_package will be true or false as it is a Boolean, but it should actually be 'present', 'absent' or 'installed'. I don't think this code will compile. I'd set variables $ensure_package and $require_package in an if statement above this resource, giving them a value based on value of $manage_package. Then thoses variables could be used below just as needed.